### ISSUE TITLE
Provide es6 modules

### ISSUE THREAD

**Issue by backspaces** [LeaVerou/bliss#226]
This is a request for an es6 module version of the project.

Generally I'd suggest simple running rollup with the rollup-plugin-commonjs which converts many legacy repos. Alas this does not work here. There is another issue request for CJS format, and if that is done, then rollup-plugin-commonjs would work.

Just to be clear, and I've guided other repos thru the process, the simplest approach is:
- Convert to modern es6+ or at least es6 import/export
- Use Rollup to convert that to UMD or IIFE formats

In other words, the root design is es6 modules, converting to any other format with Rollup. It Just Works.

**Comment by LeaVerou** [comment]
Yup, this is a huge goal of mine going forwards, especially as I want to use Bliss in ESM projects of mine!
